source/getting-started/installation-raspberry-pi-image.markdown

```diff
@@ -160,6 +160,14 @@ $ sudo apt-get upgrade
 
 Press `Y` to confirm that you would like to continue.
 
+### {% linkable_title dailout permission}
+When using USB devices users have to give dialout permission to the user `homeassistant` and restart.
```

Comment:
This should most likely be some USB devices since it's very dependent on the USB device.